Formula/libepoxy.rb

```diff
@@ -25,6 +24,7 @@ class Libepoxy < Formula
   depends_on "meson" => :build
   depends_on "ninja" => :build
   depends_on "pkg-config" => :build
+  depends_on "python" => :build
```

Comment:
should we do `python@3.9` instead of `python`?

Reply:
We should, as `python` is a moving target.

Reply:
I don't think it matters much, it just needs a `python3` interpreter at build-time.

Reply:
It matters when python migrates though

Reply:
But even then, the existing bottles will continue to work right? And afterwards when the package is rebuilt for whatever reason, the new version of python will be picked up automatically I assume?